--- MediaConch/Source/Common/Schema.cpp~	2018-04-10 13:35:59.000000000 +0200
+++ MediaConch/Source/Common/Schema.cpp	2025-08-11 22:48:48.900425195 +0200
@@ -14,6 +14,7 @@
 #include "Schema.h"
 #include <fstream>
 #include <sstream>
+#include <cstdarg>
 //---------------------------------------------------------------------------
 
 //---------------------------------------------------------------------------
